dandiapi/api/views/asset.py

```diff
@@ -122,6 +120,14 @@ class AssetViewSet(NestedViewSetMixin, DetailSerializerMixin, ReadOnlyModelViewS
     filter_backends = [filters.DjangoFilterBackend]
     filterset_class = AssetFilter
 
+    def get_queryset(self):
+        sortfield = self.request.query_params.get('sort', 'created')
+        if sortfield not in ('created', 'modified', 'path'):
+            sortfield = 'created'
```

Comment:
I think this should be an error rather than falling back to `created`.

Reply:
Done.